store/storetest/user_store.go

```diff
@@ -1855,21 +1853,21 @@ func testUserStoreGetByAuthData(t *testing.T, ss store.Store) {
 	t.Run("get by u1 auth, unknown service", func(t *testing.T) {
 		_, err := ss.User().GetByAuth(u1.AuthData, "unknown")
 		require.NotNil(t, err)
-		require.Equal(t, err.Id, store.MISSING_AUTH_ACCOUNT_ERROR)
+		require.IsType(t, &store.ErrNotFound{}, err)
```

Comment:
I'd prefer to do as `errors.As` which travels down an error chain. `IsType` would fail if the error was wrapped.

Reply:
Ok

Reply:
This is also applicable to the rest of the calls below.